pkgs/desktops/gnome/core/baobab/default.nix

```diff
@@ -26,24 +26,26 @@ stdenv.mkDerivation rec {
     sha256 = "19yii3bdgivxrcka1c4g6dpbmql5nyawwhzlsph7z6bs68nambm6";
   };
 
+  strictDeps = true;
+
   nativeBuildInputs = [
+    desktop-file-utils
+    gettext
+    glib
+    itstool
+    libhandy
```

Comment:
Could you add a comment why libhandy is here?

Reply:
It seems that compilation fails without `libhandy` here with:
```
error: Package `libhandy-1' not found in specified Vala API directories or GObject-Introspection GIR directories
```

I am still learning Nix/GNOME ecosystem so please consider my propositions as "weak".

Reply:
Hmm, the upstream source code looks normal, it probably is a bug either in some of our setup hooks, meson or vala.

So we either keep it here and add a comment ``Prevents “error: Package `libhandy-1' not found in specified Vala API directories or GObject-Introspection GIR directories”, even though it should only be a runtime dependency.”``

Or do not force `stricteDeps` for now.

Other changes look good.